.github/workflows/publish-docker-image.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         id: tags
         run: |
           if [[ '${{ inputs.pkg-type }}' = 'stable' ]]; then
-            TAGS='${{ steps.stable-tags.outputs.tag }}'
+            TAGS='${{ steps.stable-tags.outputs.tags }}'
```

Comment:
Ah, I think we will definitely need some kind of mechanism for testing the "stable" pipeline before doing the actual release. I don't see how else we would be catching these kind of errors.

Reply:
Yeah, we previously tested these jobs manually after each change.
We could do some release CI tests using a test APT repo/a test docker namespace, etc.